bin/wick-explorer

```diff
@@ -10,33 +10,48 @@
 #
 # Returns true when explorer ran successfully, false otherwise.
 wickExplorer() {
-    local explorer tempdir
+    local explorer restoreErrors result statusCode tempdir
 
     explorer="formulas/$2/explorers/$3"
+    wickStrictRun result wickFind explorer "$explorer"
 
-    if ! wickFind explorer "$explorer"; then
+    if [[ "$result" -ne 0 ]]; then
         wickError "Unable to find explorer: $explorer"
 
         return 1
     fi
 
-    # Run the explorer
     wickTempDir tempdir
 
+    #: Just like wickStrictRun, this needs to temporarily disable
+    #: error settings and traps.
+    restoreErrors="set \"-$-\";$(trap -p ERR)"
+
+    # Disable errors only for the subshell and run the explorer.
+    set +eE
+    trap "" ERR
+
     (
-        . "$explorer" > "$tempdir/stdout" 2> "$tempdir/stderr" || echo "$?" > "$tempdir/fail"
-        :
+        eval "$restoreErrors"
```

Comment:
We disable errors before we run the subshell so we can capture its return code (line 39).  Inside the subshell we enable error trapping back to what it was previously, otherwise the command may not be running in the expected strict mode.  Let's make an example.

    testMe() {
        false
        true
    }
    set -e
    wickStrictRun testMe

If I don't enable strict mode again in the subshell, `testMe` will always return success.

As for the bit about disabling errors again after the command ... we don't need to do that because the subshell context is finished and we go back to whatever settings were set in the parent.  I could turn on DEBUG traps and all sorts of crazy things in the subshell.  When we exit the subshell, all of that is reset.